lib/hammer_cli_foreman_remote_execution/options/normalizers.rb

```diff
@@ -1,3 +1,5 @@
+require 'parse-cron'
```

Comment:
Well I thought so, and I do like it, is there a reason not to do it? Most input that can be validated at the hammer side usually is - dates, lists, etc.

I can remove it if you really don't like it.